net-p2p/qbittorrent/qbittorrent-9999.ebuild

```diff
@@ -47,7 +47,9 @@ RDEPEND="
 		dev-qt/qtnetwork:5[ssl]
 		dev-qt/qtsql:5[sqlite]
 		dev-qt/qtxml:5
-	)"
+	)
+	>=acct-group/qbittorrent-0
+	>=acct-user/qbittorrent-0"
```

Comment:
Seems a little bit overly verbose. A single line containing `act-user/qbittorrent` would be sufficient.

Reply:
fixed